language/evm/move-to-yul/tests/ControlStructures.exp.apply-cfg-to-scf

```diff
@@ -0,0 +1,370 @@
+                    {
+                        // label L2
+                        // $t4 := 0
+                        $t4 := 0
+                        // $t5 := ==($t0, $t4)
+                        $t5 := $Eq(x, $t4)
+                        // if ($t5) goto L4 else goto L3
+                        switch $t5
+                        case 0  { $block := 4 }
+                        default { $block := 5 }
+                    }
+                    {
+                        // label L3
+                        // $t1 := 1
+                        $t1 := 1
+                        // $t2 := >=($t0, $t1)
+                        $t2 := $GtEq(x, $t1)
+                        // if ($t2) goto L0 else goto L2
+                        switch $t2
+                        case 0  { $block := 3 }
+                        default { $block := 2 }
+                    }
+                    {
+                        // label L2
+                        // $t4 := 0
+                        $t4 := 0
+                        // $t5 := ==($t0, $t4)
+                        $t5 := $Eq(x, $t4)
+                        // if ($t5) goto L4 else goto L3
+                        switch $t5
+                        case 0  { $block := 4 }
+                        default { $block := 5 }
+                    }
+                }
+            }
+
+            function A2_M_h1(x) -> $result {
+                let tmp_$1, $t2, $t3, $t4, $t5
+                {
+                    // $t2 := 0
+                    $t2 := 0
+                    // $t3 := >($t0, $t2)
+                    $t3 := $Gt(x, $t2)
+                    // if ($t3) goto L0 else goto L2
+                    switch $t3
+                    case 0  { $block := 3 }
+                    default { $block := 2 }
+                }
+                {
+                    // label L0
+                    // $t4 := 1
+                    $t4 := 1
+                    // $t1 := $t4
+                    tmp_$1 := $t4
+                    // goto L3
+                    $block := 5
+                }
+                {
+                    // label L2
+                    // $t5 := 2
+                    $t5 := 2
+                    // $t1 := $t5
+                    tmp_$1 := $t5
+                    // goto L3
+                    $block := 5
+                }
+                {
+                    // label L3
+                    // return $t1
+                    $result := tmp_$1
+                    leave
+                }
+            }
+
+            function A2_M_h2(x) {
+                let $t1, $t2, $t3
+                {
+                    // $t1 := 0
+                    $t1 := 0
+                    // $t2 := >($t0, $t1)
+                    $t2 := $Gt(x, $t1)
+                    // if ($t2) goto L0 else goto L2
+                    switch $t2
+                    case 0  { $block := 3 }
+                    default { $block := 2 }
+                }
+                {
```

Comment:
This is somehow wrong. The Abort block is never reached. Applies to other examples as well.